frontend/app/src/components/accounts/AccountForm.vue

```diff
@@ -399,12 +399,11 @@ export default class AccountForm extends Vue {
   payload(): BlockchainAccountPayload {
     let xpubPayload: XpubPayload | undefined;
     if (this.isBtc && this.isXpub) {
-      const xpubKey = this.xpub.trim();
-      const xpub = AccountForm.isPrefixed(xpubKey)
-        ? xpubKey
-        : `${this.xpubKeyType}${xpubKey}`;
+      const trimmedKey = this.xpub.trim();
```

Comment:
So just to make sure I understand what this does.

If I add an `xpub3XDSADSADSADSADSADSADADASDSA` as WPKH then this will change it to `zpub3XDSADSADSADSADSADSADADASDSA` ?

Reply:
yup :)

Reply:
But the copy paste auto-detection still works?

Reply:
yup, when you paste it should change to what is inferred

Reply:
Okay then should be what I asked. But still not 100% sure this is what the users would need. We would need more feedback for the various types of wallets and xpubs. Still this should go in now.